lib/taiko.js

```diff
@@ -2737,6 +2737,9 @@ module.exports.evaluate = async (selector, callback, options = {}) => {
       returnByValue: true,
     });
   });
+  if (options.omitDescription !== true) {
+    descEvent.emit('success', 'Evaluated given script');
+  }
   descEvent.emit('success', 'Evaluated given script');
```

Comment:
Is this line needed as there is an if condition just before it?

Reply:
Yes @zabil . So we are validating if the description is already present then don't print an unnecessary message because
all the taiko APIs prints a description of the action they performed.

Reply:
@zabil I removed the below line which is outside the "if" condition.